service/awsconfig/v8/resource/cloudformation/validate.go

```diff
@@ -43,7 +43,7 @@ func (r *Resource) validateHostPeeringRoutes(cluster v1alpha1.AWSConfig) error {
 		},
 	}
 	output, err := r.hostClients.EC2.DescribeRouteTables(input)
-	if err == nil && len(output.RouteTables) == 1 {
+	if err == nil && len(output.RouteTables) >= 1 {
```

Comment:
Why not comparing it to `2` instead?

Reply:
+1 

I feel like this is in general super implicit and magic. How can we prevent this in the future? How can this number become a one or a three?

Reply:
Ok, I think I do not understand this. When you think this is right. Go ahead.

Reply:
Ack. Makes sense.
Nitpick: in that case natural check is `len(output.RouteTables) > 0`.